monai/config/deviceconfig.py

```diff
@@ -23,6 +23,8 @@
 import torch
 
 import monai
+from monai.utils.deprecate_utils import deprecated
+from monai.utils.enums import IgniteInfo as NewIgniteInfo
```

Comment:
I would suggest changing the name of `NewIgniteInfo` to `_IgniteInfo`. Not a blocker though.

Reply:
Updated, thanks.